protypes/Eclipse GUI with Wizards and Dialogs/src/BeagleAnalysisWizard.java

```diff
@@ -0,0 +1,64 @@
+
+import org.eclipse.jface.dialogs.MessageDialog;
+import org.eclipse.jface.wizard.Wizard;
+import org.eclipse.swt.widgets.Shell;
+
+/**
+ * The class setting up and controlling the wizard.
```

Comment:
> the wizard

which one? Harry Potter? ;)

No seriously, that might not be clear to the reader, all though it is for us in the moment.